bcsdlab_page/src/components/SpaComponents/MainPage/Components/TrackSection.vue

```diff
@@ -76,14 +76,19 @@
 
       async getTotalTrackInfo() {
 
-        for(let i=0; i<5; i++) {
-          let track = await api.getTrackInfo(i+1);
-          console.log(track);
-          console.log(track.data.Members.length)
-          this.trackDatas[i].memberCount = String(track.data.Members.length).padStart(2,0);
+        for(let i=0; i<4; i++) {
+            let track = await api.getTrackInfo(i+1);
+            console.log(track);
+            console.log(track.data.Members.length)
+            this.trackDatas[i].memberCount = String(track.data.Members.length).padStart(2,0);
         }
+        if(this.trackDatas[4].name === "UI/UX"){
+          let track = await api.getTrackInfo(7);
+          console.log(track);
+          return this.trackDatas[4].memberCount = String(track.data.Members.length).padStart(2,0);
+          }
```

Comment:
```suggestion
      // FE, BE, Android, Game, UI/UX
        [1, 2, 3, 4, 7].forEach((i, idx) => {
            let track = await api.getTrackInfo(i);
            this.trackDatas[idx].memberCount = String(track.data.Members.length).padStart(2,0);
        })
```
이런 문법으로도 적을 수 있다는 점을 알아두면 좋을듯!

Reply:
저 경우에 idx에는 i와 다르게 1부터 5가 할당이 되는건가요?

Reply:
`map`이나 `forEach`의 두번째 인자인 index는 0부터 1씩 올라가는 수입니당

Reply:
아하 알겠습니다ㅎㅎ 수정해서 다시 올리겠습니다